docs/pbc/all/cart-and-checkout/approval-process-feature-overview.md

```diff
@@ -18,6 +18,14 @@ The *Approval Process* feature lets B2B customers have multiple people contribut
 
 Permissions related to the approval process are configured based on the restrictions applied to a [company role](/docs/scos/user/features/{{site.version}}/company-account-feature-overview/company-user-roles-and-permissions-overview.html). Generally, the approval process is initiated when the cart total exceeds a certain amount set in the *Buy up to grand total* permissions. For example, an employee in a company may have to send their order to the manager for approval if the total order cost is above a certain amount. Only after the manager has received the request and approved the order, the employee can proceed to the checkout.
 
+{% info_block warningBox "Note" %}
```

Comment:
Come up with a name for this note

Reply:
@andriitserkovnyi Not sure about the title. We can cut out and paste the first sentence:

```suggestion
{% info_block warningBox "Approvers can only approve orders within their own business unit" %}
```

Reply:
```suggestion
{% info_block warningBox "Approvals within a business unit" %}
```